frontend/src/views/ScheduleView.vue

```diff
@@ -87,7 +87,6 @@ const schedules = ref([]);
 const firstSchedule = computed(() => (schedules.value?.length > 0 ? schedules.value[0] : null));
 const schedulesReady = ref(false);
 const getFirstSchedule = async () => {
-  calendarEvents.value = [];
```

Comment:
I didn't find any reason to keep this here. This was the cause for calendar events to disappear on schedule save.